src/test/java/com/alipay/oceanbase/rpc/ObTableClientTest.java

```diff
@@ -2489,4 +2489,54 @@ PRIMARY KEY (`c1`)
             client.delete("test_timestamp_table", new Object[] { "key_0" });
         }
     }
+
+    @Test
+    public void testQueryWithEmptyTable() throws Exception {
+        // test query with empty table name.
+        /*
+         CREATE TABLE `test_query_filter_mutate` (`c1` bigint NOT NULL, `c2` varbinary(1024) DEFAULT NULL,
+                                                  `c3` varchar(20) DEFAULT NULL,`c4` bigint DEFAULT NULL, PRIMARY KEY(`c1`))
+                                                  partition by range columns (`c1`) ( PARTITION p0 VALUES LESS THAN (300),
+                                                  PARTITION p1 VALUES LESS THAN (1000), PARTITION p2 VALUES LESS THAN MAXVALUE);
+         */
+        final String TABLE_NAME = "test_query_filter_mutate";
+        System.setProperty("ob_table_min_rslist_refresh_interval_millis", "1");
+
+        ((ObTableClient) client).addRowKeyElement(TABLE_NAME, new String[] { "c1" }); 
```

Comment:
why set tableName equals to test_query_filter_mutate and addRowKeyElement for table test_query_filter_mutate, it  seems a test just for empty table?

Reply:
already edited